config/parameters/truth/detumble.txt

```diff
@@ -0,0 +1,26 @@
+# Initial conditions for simulations starting during detumble. These initial
+# conditions were generated with the following command:
+#   python -m psim -s 10000 -c truth/base,truth/deployment --snapshot snapshot.txt DualAttitudeOrbitGnc
+#
+
+# Time model initial conditions.
+
+truth.t.ns  1700000000000
```

Comment:
This parameter will change if the flight software cycle time changes (which is possible.) Tracked in https://github.com/pathfinder-for-autonomous-navigation/FlightSoftware/issues/532

Reply:
Thanks - and yeah @shihaocao will be keeping track of that for now. #266 will help make changes possible from the PTest side as well.